blssignatures/helpertypes.go

```diff
@@ -0,0 +1,11 @@
+package blssignatures
+
+type indexedSig struct {
+	index int
+	sig   []byte
+}
+
+type aggSigData struct {
+	Sig  []byte
+	Mask []byte
+}
```

Comment:
I think this can just be removed and only the `sig []byte` be used. `granite.go` already keeps track of the signatures being aggregated in the `signers bitfield.Bitfield` of `QuorumSignature`. Both are redundant, and I do not see a strong reason to pair together in the serialization of the signature the list of signers. if we need a `sign.Mask` to make calls to the `bdn` package, we can just locally create it. Specially if `sign.Mask` does not serialize as efficiently as bitfield.Bitfield does with RLE.

Reply:
Also, keep in mind that the list of signers should be easily accessible from Granite's `quorumState` to check for the existence of a strong quorum in the aggregated signature, and this need not be necessarily performed before the deserialization/verification of the aggregated signature. 

So I would rather a verification interface that explicitly takes the list of signers than one that keeps them serialized within the aggregated signature. The bitfield of signers comes sorted by voting power anyways so it should be ok for verification/aggregation AFAIK.